src/styles/index.scss

```diff
@@ -1,3 +1,111 @@
 body {
   margin: 0;
 }
+
+.card {
+  max-width: 200px;
+  height: 406px;
+
+  box-sizing: border-box;
+
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+
+  position: relative;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.card__picture {
+  position: absolute;
+  top: 33px;
```

Comment:
You don't need to use position absolute in this design. You can solely rely on flex